src/main/java/com/codeplanks/home360/auth/RegisterRequest.java

```diff
@@ -0,0 +1,50 @@
+package com.codeplanks.home360.auth;
+
+
+import com.codeplanks.home360.validation.Password;
+import com.codeplanks.home360.validation.ValidEmail;
+import jakarta.persistence.Column;
+import jakarta.validation.constraints.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class RegisterRequest {
+
+    @NotNull(message = "First name is required")
+    @Column(name = "firstName", nullable = false)
```

Comment:
This class is not an Entity i.e table, there is no need for the Column annotation I believe